cwms_radar_api/src/main/java/cwms/radar/data/dao/OfficeDao.java

```diff
@@ -6,36 +6,53 @@
 import cwms.radar.data.dto.Office;
 import org.jooq.DSLContext;
 import org.jooq.Record4;
+import org.jooq.Table;
 
 import usace.cwms.db.jooq.codegen.tables.AV_OFFICE;
+import usace.cwms.db.jooq.codegen.tables.AV_LOC2;
 
 public class OfficeDao extends JooqDao<Office>
 {
```

Comment:
Please fix the formatting on these brackets while you're in the file. (Normally I wouldn't worry about it but I'm already asking to to tweak a bunch of other formatting.

Reply:
I went through and fixed all formatting errors that `gradlew checkStyleMain` returned for the two files i've touched, including the javadoc errors.